rdt-open-reader/src/main/java/io/ona/rdt/interactor/PatientProfileActivityInteractor.java

```diff
@@ -1,9 +1,9 @@
 package io.ona.rdt.interactor;
 
-import io.ona.rdt.util.FormSaver;
+import io.ona.rdt.util.CovidFormSaver;
 
 /**
  * Created by Vincent Karuri on 17/02/2020
  */
-public class PatientProfileActivityInteractor extends FormSaver {
+public class PatientProfileActivityInteractor extends CovidFormSaver {
```

Comment:
Any reason for doing this?

Reply:
I think you can make `CovidPatientProfileActivityInteractor` extend `CovidFormSaver` instead.

Reply:
`PatientProfileActivityPresenter.createInteractor()` needs to return `PatientProfileActivityInteractor`